src/helpers/xml-builder.js

```diff
@@ -483,22 +499,19 @@ const fixupMargin = (marginString) => {
     const matchedParts = marginString.match(inchRegex);
     return inchToTWIP(matchedParts[1]);
   } else if (percentageRegex.test(marginString)) {
-    // This requires changes in lot of functions. So, for now, we are returning the percentage value as it is.
-    // TODO: Revisit this and see how margins in percentages are handled and change in respective functions.
     const matchedParts = marginString.match(percentageRegex);
-    return matchedParts[1]
+    return matchedParts[1];
   }
 };
 
 const borderStyleParser = (style) => {
-  // Accepted OOXML Values for border style: http://officeopenxml.com/WPtableBorders.php
```

Comment:
please do not remove comments? Is this all coming from AI which removes comments?

Reply:
I don't know, when I pushed it wasn't like that
I'll change that